bin/Cores/cores.json

```diff
@@ -722,6 +738,7 @@
   },
   "virtualjaguar_libretro":{
     "name": "Virtual Jaguar",
+    "extensions": "j64|rom"
```

Comment:
First, this line _must_ end with a comma or the parser stops. I couldn't even see the core until I fixed that.

Secondly, this line is only necessary if we want to change the list of extensions supported by the core (mostly to limit the supported files to those associated with the console - i.e. `gbc` for console 6 because the core also supports console 4 and maybe 5).

The core reports that it supports `j64|jag|zip`. You've misplaced the `jag` support (all of my dumps happen to be zipped .jag files).

The [documentation](https://docs.libretro.com/library/virtual_jaguar/#extensions) suggests that in addition to `j64`, `jag`, and `rom`, the core also supports `abs`, `cof`, `bin`, and `prg` extensions, so it's weird that the core doesn't report any of those.

At a minimum, you need to add a comma and support for the jag extension. I'm not sure what the implications are for supporting the other extensions. Do we need to worry about headers on any of them?

Reply:
http://freejag.atari.org/forum/index.php?topic=59.0;wap2
Per that, jag/j64 has an 8KiB header, rom doesn't, and the rest are primarily homebrew formats (and thus probably headerless).

Reply:
[This](https://forums.atariage.com/topic/162949-file-headers/?do=findComment&comment=2012279) seems to be the only thing I can find documenting the header format.

My `.jag` files appear to be headerless (the first 600 or so bytes appear to be random binary). It's probably related to this comment:

> Note that as of this writing (1/1/2004) that the GoodJag ROM renaming tool by Cowering incorrectly renames Jaguar cartridge ROM dumps to have a .jag extension

Reply:
[This](https://forums.atariage.com/topic/162949-file-headers/?do=findComment&comment=2012104) is slightly concerning. 

> A "cart" (usually a .jag but should really be a .j64) loads at 0x800000, and the start address (usually 0x802000) is to be found at 0x800404.
> A .rom file is a cart minus the header, loads and runs at 0x802000.

My `.jag` file does indeed have a start address of 0x802000 (big endian) at 0x800404:
![image](https://user-images.githubusercontent.com/32680403/193033462-a4609eaf-7c3b-4aae-a9dc-55f692e3aee0.png)

So if I had a `.rom` cart of the same game, would it be 8KiB shorter? In which case, it would not generate the same hash. @televandalist - are your `.rom` files generating the same hashes already linked to the games?